conda/environments/cuxfilter_dev_cuda10.0.yml

```diff
@@ -1,13 +1,13 @@
 # name: cudf_dev10.0
 channels:
-- rapidsai
+- rapidsai-nightly
```

Comment:
You should leave `rapidsai` in this file above `rapidsai-nightly`. For example, this is how cudf does it: https://github.com/rapidsai/cudf/blob/branch-0.13/conda/environments/cudf_dev_cuda10.0.yml

Reply:
updated it, thanks!